packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/ManagedACTAccessRequirementItemView.tsx

```diff
@@ -0,0 +1,305 @@
+import { formatDate } from '@/utils/functions/DateFormatter'
```

Comment:
File contains the View-only logic that was originally in `ManagedACTAccessRequirementItem.tsx`. The few meaningful changes I made are called out.